DigitalLearningSolutions.Web/ViewModels/MyAccount/MyAccountViewModel.cs

```diff
@@ -55,13 +56,17 @@ string switchCentreReturnUrl
             AllCentreSpecificEmails = allCentreSpecificEmails;
             DlsSubApplication = dlsSubApplication;
             SwitchCentreReturnUrl = switchCentreReturnUrl;
+            PrimaryEmailIsUnverified = userAccount.EmailVerified == null;
+            UnverifiedCentreEmails = unverifiedCentreEmails;
         }
 
         public int? CentreId { get; set; }
         public string? CentreName { get; set; }
 
         public string PrimaryEmail { get; set; }
 
+        public bool PrimaryEmailIsUnverified { get; set; }

```

Comment:
Rider's telling me it can't be readonly :/

Reply:
Is it telling you why?

Reply:
Weirdly no, even when I remove all usages of it, it still says it can't be readonly. Only usages are in the constructor and passing it into the verify-your-email-warning-view-component

Reply:
Oh, I meant readonly as in not settable rather than necessarily adding a `readonly`.
In c# you can either have a `public readonly` field or a public property with no setter ([docs here](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/members)). The more common policy on this project to just remove the `set` accessor. Could you do that?

Reply:
(When Rider says, 'Property cannot be "readonly"', it doesn't mean, 'This property cannot be "readonly"' - it means that "readonly" properties aren't a thing in c#.)

Reply:
Ohhhh I see, yup removed :)